mutis/correlation.py

```diff
@@ -51,20 +51,25 @@ def __init__(self, signal1, signal2, fcorr):
         self.tmax_valid = +(np.max([t1.max() - t1.min(), t2.max() - t2.min()]) - np.min([t1.max() - t1.min(), t2.max() - t2.min()]))/ 2 + self.t0_full
 
     def gen_synth(self, samples):
-        """Description goes here.
+        """Generates the synthethic light curves.
 
+        Generates the specified number `samples` of synthethic light curves for each signal, to be used to compute the significance the correlation.
+        
         Parameters
         ----------
         samples : :py:class:`~int`
-            Parameter description.
+            Number of synthethic light curves to be generated for each signal.
         """
 
         self.samples = samples
         self.signal1.gen_synth(samples)
         self.signal2.gen_synth(samples)
 
     def gen_corr(self):
-        """Description goes here."""
+        """Generates the correlation of the signals.
+        
+        Generates the correlation of the signals, and computes their confidence level from the synthethic light curves, which must have been generated before.
```

Comment:
Please, do not make a single-line docstring, add breaks around char 70, it makes the whole thing more readable.